packages/api/src/index.ts

```diff
@@ -84,6 +87,7 @@ passport.use(
               const newUser = new User({
                 name: profile.username,
                 githubId: profile.id,
+                githubUsername: profile.username,
```

Comment:
We are we saving `profile.username` twice? If it's the same value, let's either rename the original or use that where you're using `githubUsername`

Reply:
We need both field because one field is the name we use for our application (their profile page, future banners, etc).  We currently allow them to change this name so if their github username is something nonsensical they can change their name in our system.  We need to know their actual githubUsername as well becuase graphql only retruns usernames, it does not return githubIds.  So we need to know the username to match up contributions to the correct user.  We update this username everytime they log in on line 90 or 106, becuase it is possible for them to update thier githubUsername.